src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -600,10 +600,20 @@ def utask_main(uworker_input):
 
 def utask_postprocess(output):
   """Postprocess in a trusted bot."""
-  if output.error is not None:
+  if output.error > 0:
     uworker_handle_errors.handle(output, HANDLED_ERRORS)
     return
 
+  # Since uworker_input is set automagically, this should never be true
+  if output.uworker_input.ByteSize() == 0:
+    raise errors.BadStateError
+
+  # At this point, minimize_task_output should be set and
+  # last_crash_result_dict should not be empty
+  if output.minimize_task_output.ByteSize() == 0 \
```

Comment:
Minimize_task_output wont necessarily be set when there's an error in setup.

Reply:
if theres an error in set up, we shouldnt get to this point, no?